x-pack/plugins/observability_solution/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -7,29 +7,32 @@
 
 import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 import { i18n } from '@kbn/i18n';
-import { AlertsLocatorParams, getAlertDetailsUrl } from '@kbn/observability-plugin/common';
+import { getAlertDetailsUrl } from '@kbn/observability-plugin/common';
 import {
   ALERT_CONTEXT,
   ALERT_EVALUATION_THRESHOLD,
   ALERT_EVALUATION_VALUE,
   ALERT_REASON,
+  ALERT_ACTION_GROUP,
 } from '@kbn/rule-data-utils';
 import { ElasticsearchClient, IBasePath } from '@kbn/core/server';
 import {
   ActionGroup,
   ActionGroupIdsOf,
-  Alert,
   AlertInstanceContext as AlertContext,
   AlertInstanceState as AlertState,
-  RuleExecutorServices,
   RuleTypeState,
+  RuleExecutorOptions,
+  AlertsClientError,
 } from '@kbn/alerting-plugin/server';
-import { LocatorPublic } from '@kbn/share-plugin/common';
 import { addSpaceIdToPath } from '@kbn/spaces-plugin/common';
 import { asyncForEach } from '@kbn/std';
+import { ObservabilityMetricsAlert } from '@kbn/alerts-as-data-utils';
```

Comment:
this should be `ObservabilityLogsAlert`?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/178680/commits/caa163f3d5455b6a09160abc491d9a870fcd208f